### ISSUE TITLE
Unpredictable Conversation Rewards

### ISSUE THREAD

**Issue by pavlovcik** [ubiquity/ubiquibot#673]
- The conversation rewards should not have been granted to the assignee (me)
- The amount seems way higher than expected. 
- I need to be able to understand why exactly it came to the numbers it did while I calibrate the comment incentives globally and for debugging. Can this be logged?

I have a hypothesis that `monospace` words are extremely heavily credited. Update: I just regenerated the rewards and removed the `monospace` it only was credited as $5. We should update those to $1 but make 
```
code blocks 
```

worth $5.

---

### [ **diamondnegroni: [ CLAIM 2.7 WXDAI ]** ](https://pay.ubq.fi?claim=eyJwZXJtaXQiOnsicGVybWl0dGVkIjp7InRva2VuIjoiMHhlOTFEMTUzRTBiNDE1MThBMkNlOERkM0Q3OTQ0RmE4NjM0NjNhOTdkIiwiYW1vdW50IjoiMjcwMDAwMDAwMDAwMDAwMDAwMCJ9LCJub25jZSI6IjEwNTQwNzQ4MDkwNjM0NzYyMDAzMDMyMzUxOTA3MTk1NjQ3NDg2ODI4MTczODQ1ODUwMjc0NjgwOTM3NTU0MTU2NTY0NDQ1ODExOTYyNiIsImRlYWRsaW5lIjoiMTE1NzkyMDg5MjM3MzE2MTk1NDIzNTcwOTg1MDA4Njg3OTA3ODUzMjY5OTg0NjY1NjQwNTY0MDM5NDU3NTg0MDA3OTEzMTI5NjM5OTM1In0sInRyYW5zZmVyRGV0YWlscyI6eyJ0byI6IjB4QjQ5Yzg5MUMxZTFFNjkzOTU4ZDExRjI3YTkyMEZlZTM0M2FFYjY4ZCIsInJlcXVlc3RlZEFtb3VudCI6IjI3MDAwMDAwMDAwMDAwMDAwMDAifSwib3duZXIiOiIweGY4N2NhNDU4M0M3OTIyMTJlNTI3MjBkMTI3RTdFMEEzOEI4MThhRDEiLCJzaWduYXR1cmUiOiIweGUxZjUzODUyZGVlYjk5NjExMTUxOTVmNTVjNmMwYTk3NzFiNjZjODU4ZGU3MDU0OWExNWVjMzg3YTIwOTM4NjQ1NmQ2YTRiYjAxZjc5Mjk3NzlmMjRlN2NkNWIyNzEyMjEwYzk4MGFhYWFkOTdjNTVhNGEwYThkOTQwZGYxZWU2MWIifQ==&network=100)

_Originally posted by @ubiquibot[bot] in https://github.com/ubiquity/sponsorships/issues/4#issuecomment-1694494805_

**Comment by pavlovcik** [comment]
I did some research [here](https://github.com/ubiquity/ubiquibot/pull/674#issuecomment-1694505668) and realized that we must use `raw` and we must convert that into HTML.

**Comment by whilefoo** [comment]
But if we want to use HTML tags in the config than we must convert the raw Markdown syntax to HTML. [This](https://github.com/ubiquity/ubiquibot/issues/496) task was about moving from Markdown to HTML

**Comment by pavlovcik** [comment]
> But if we want to use HTML tags in the config than we must convert the raw Markdown syntax to HTML. [This](https://github.com/ubiquity/ubiquibot/issues/496) task was about moving from Markdown to HTML

It was supposed to be for unambiguous config property names.

**Comment by whilefoo** [comment]
I see, this was misunderstanding on my end. So we should parse the comment in Markdown format and convert HTML tags in the config to the Markdown equivalent?

**Comment by pavlovcik** [comment]
Yes I just figured that in the config it's much clearer to use HTML. So it's easier to config. 

As we see now the comment incentive calculator must consume the raw source code or else this won't work. 

The raw source code is probably going to be markdown. 

What happens if it is html? Will mdast handle this?

**Comment by whilefoo** [comment]
Mdast will leave html as it is.

**Comment by EtherealGlow** [comment]
@whilefoo are you still working on this
